### ISSUE TITLE
White noise

### ISSUE THREAD

**Issue by Psychokiller1888** [respeaker/seeed-voicecard#71]
Hi!

I'm using respeaker 4 and respeaker 2 on many devices, on raspi 3B, 3B+ and pi zero.... On every devices the sound quality is horrible, full of white noise, hard to understand what I'm saying... Is there a way to obtain a good sound quality?

**Comment by Psychokiller1888** [comment]
I can't, I'm using the devices as voice assistants

**Comment by Psychokiller1888** [comment]
This is respeaker 2, not too bad, but not good either
https://soundcloud.com/laurent-chervet-psychokiller1888/test3

Respeaker 4 is far worse than that, will upload a sound test

**Comment by Pillar1989** [comment]
@Psychokiller1888 any update ? I did not see your 4mic hat sound test file.

**Comment by Psychokiller1888** [comment]
Oh, yeah, there you go :)

https://soundcloud.com/laurent-chervet-psychokiller1888/respeaker4

**Comment by Psychokiller1888** [comment]
Any update? I don't see any reaction since I shared the audio?

**Comment by Pillar1989** [comment]
@Psychokiller1888 Sorry, we just passed China's labor day, back to your issue now. Do you modify the alsamixer's setting?

**Comment by Psychokiller1888** [comment]
No, it's not modified

```The IPC key of dmix or dsnoop plugin must be unique
# If 555555 or 666666 is used by other processes, use another one

pcm.!default {
    type asym
    playback.pcm "playback"
    capture.pcm "ac108"
}

pcm.playback {
    type plug
    slave.pcm "hw:0,0"
}

pcm.dmixed {
    type dmix
    slave.pcm "hw:0,0"
    ipc_key 555555
}

pcm.ac108 {
    type plug
    slave.pcm "multiapps"
}

pcm.multiapps {
    type dsnoop
slave.pcm "hw:1,0"
    ipc_key 666666
}
```

**Comment by bkchr** [comment]
@Psychokiller1888 could you solve this problem in any way? I have the exact same problem and getting crazy that this is not fixable :(

**Comment by Psychokiller1888** [comment]
Nope.... Seeed and @Pillar1989 don't care about this problem and won't solve it. Get rid of your respeaker 4 and get something more reliable, you can't fix it

**Comment by bkchr** [comment]
Did you found any microphone without noise? I already tried several USB microphones and the respeaker 2, but all have some sort of noise :(

**Comment by Psychokiller1888** [comment]
well, on a matrix voice or creator you're better qualitywise. Depends a lot on what you want to do

**Comment by bkchr** [comment]
Just want to have a microphone that can record without any background noise. Don't need it for any voice activation stuff, just sound recording.

**Comment by Psychokiller1888** [comment]
I don't know then, I'm not using it this way, I'm using it for voice assistants

**Comment by bkchr** [comment]
Okay, thanks for your help :)

**Comment by bkchr** [comment]
@Psychokiller1888 One last question, do you use Matrix Voice and have better results regarding noise?

**Comment by Psychokiller1888** [comment]
sure, matrix voice is light years over respeaker
